### ISSUE TITLE
Point by point data for 2016

### ISSUE THREAD

**Issue by silkysmoooth** [JeffSackmann/tennis_pointbypoint#9]
Hi,
Can't seem to find pbp data for 2016 anywhere. Does it exist ?
Thanks

**Comment by majki37** [comment]
Yes this year is missing, which is very unfortunate if you wand analysis over more years.
